README.md

```diff
@@ -1,54 +1,94 @@
 
-After installing docker on your machine, follow these steps in order to run the project:
-1. **Start docker (on windows, you may simply launch docker desktop).**
+## Getting Started
 
-2. **In order to test if docker is currently running on your machine, run the following command:** `docker ps`\
-This command will show you all the docker containers that are currently running on your machine. Since no docker containers are currently running, you will simply see the following output in the command line:\
-**CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES**
+### Prerequisites
 
-3. **In order to run all three docker containers, run the following command:** `docker-compose up`\
-If you wish to run a specific container only, you may run the following command: `docker-compose up CONTAINER`\
-where CONTAINER is the name of the specific container which can either be frontend, backend, or database.
-Note that when you choose to run the backend container only, the database container will also be started since it is required in order for the backend to function properly.
+* Install [Docker](https://www.docker.com/)
 
-4. **In order to open the website, enter the following link in your browser:** `http://localhost:3000/`
+### Installation
 
-5. **In order to shut down all running docker containers, run the following command:** `docker-compose stop`\
-If you wish to stop a specific container, you may run the following command: `docker-compose stop CONTAINER`\
-where CONTAINER is the name of the specific container which can either be frontend, backend, or database.
+1. Request access to the [environment repository](https://github.com/SOEN-390-TEAM09/env/tree/c1a1a039ba015fd87d5b70c1c31ca5a1069f37bc)
+2. Clone the repo with the submodules included
+    ```
+    git clone https://github.com/Talos10/SOEN390-team-9.git --recursive
+    ```
 
-#### Important additional information:
+### Running using docker
 
-When running a docker container for the first time, the required dependencies are automatically downloaded and cached such that subsequent runs of the containers will be faster. In the case that there are changes in the dependencies (i.e. the case where you would run the command npm install), the docker containers will need to be rebuilt. In order to do that, simply run the following command: `docker-compose up --build`\
-This command will rebuild all docker containers (and thereby update their dependencies). If you wish to rebuild a specific container only, you may run the following command: `docker-compose up --build CONTAINER`\
-where CONTAINER is the name of the specific container which can either be frontend, backend, or database.
+* Start Supreme ERP
+    ```
+    docker-compose up
+    ```
+* OR start a single service
+    ```
+    docker-compose up (backend or frontend or database)
+    ```
+* Default credentials: `admin@email.com` for the email and `admin` for the password.
+* Frontend is hosted on http://localhost:3000
```

Comment:
Can you make it more explicit that the TA will have to go to localhost:300 to have access to the app

Reply:
It's in the front page 🐒

Reply:
I know that it is on the frontpage, jsut make sure he clicks on the right one lol